### ISSUE TITLE
Insure more a11y compliance

### ISSUE THREAD

**Issue by behivetech** [jamesmfriedman/rmwc#553]
- What RMWC Version are you using [major.minor.patch]:
5.7.2
- Name your build system [Webpack, Rollup...]:
Gatsby 2.19.16
- Describe the bug with as much detail as possible:
TextField component doesn't add aria-invalid when the invalid prop is set to true.
- What happened, and what was supposed to happen:
When the invalid prop is set to true, there should be an aria-invalid attribute set to true on the `<input />`.

Also, I haven't really dug into the code, but it would be nice to try to get this library more a11y compliant as opposed to having to add the aria-* props manually. Maybe run an a11y scanner of some sort on the components to see what might be missing.

**Comment by jamesmfriedman** [comment]
Love the idea. It does use as many of the recommended ones from Google as possible, but there are definitely areas lacking. Any help in this area would be welcomed.

**Comment by bruqui** [comment]
For sure... I was going to mention I'll try to help. I'm pretty tight on time right now because of the amount of work on my plate; however, I'll hit you up on Discord if I have some time to work on it and maybe make a game plan.

**Comment by jamesmfriedman** [comment]
Taking this out of the current milestone, but please add any additional things you find on a11y that can be improved.
